src/templates/partials/home-product.hbs

```diff
@@ -4,11 +4,11 @@
     <p>{{product.description}}</p>
     <div class="shop-now">
       <span>Shop now</span>
-      <svg viewBox="0 0 24 24">
+      <svg viewbox="0 0 24 24">
         <path d="M12 4l-1.41 1.41L16.17 11H4v2h12.17l-5.58 5.59L12 20l8-8z" />
         <path d="M0 0h24v24H0z" fill="none" />
       </svg>
     </div>
   </div>
-  <img class="lazy" crossorigin="anonymous" data-id="{{product.id}}" alt="{{product.name}}" srcset=""></img>
-</a>
\ No newline at end of file
+  <img class="lazy" data-id="{{product.id}}" alt="{{product.name}}" srcset="">
```

Comment:
`crossorigin` attribute should remain, I think.

Reply:
Oops — not sure what happened there :(. Reverted.